src/WebJobs.Extensions.DurableTask/Bindings/OrchestrationTriggerAttributeBindingProvider.cs

```diff
@@ -172,6 +172,7 @@ public Task<ITriggerData> BindAsync(object? value, ValueBindingContext context)
                         NewEvents = { remoteContext.NewEvents.Select(ProtobufUtils.ToHistoryEventProto) },
                         EntityParameters = remoteContext.EntityParameters.ToProtobuf(),
                     };
+                    orchestratorRequest.Properties.Add(ProtobufUtils.ConvertPocoToProtoMap(remoteContext.Configurations));
```

Comment:
Non-blocking: I'm not sure I understand why we need to convert from a POCO to a map. Wouldn't it make more sense to convert from a POCO to a custom protobuf message type? Was it a mistake to use a map in the protobuf contract? What we're doing now seems like it's creating unnecessary overhead, especially now that we have to use reflection like this for every orchestrator request.